packages/ui-components/src/components/UIInput/UITextInput.tsx

```diff
@@ -154,11 +153,9 @@ export class UITextInput extends React.Component<UITextInputProps> {
                         fontWeight: 'normal',
                         boxSizing: 'border-box',
                         borderRadius: COMMON_INPUT_STYLES.borderRadius,
-                        fontFamily: 'var(--vscode-font-family)',
```

Comment:
and here?

Reply:
as checked - I do not see reason to leave it, same explanation as - https://github.com/SAP/open-ux-tools/pull/3060#discussion_r2020847135